browser/ui/toolbar/brave_location_bar_model_delegate.cc

```diff
@@ -6,27 +6,50 @@
 #include "brave/browser/ui/toolbar/brave_location_bar_model_delegate.h"
 
 #include "base/strings/utf_string_conversions.h"
+#include "brave/common/url_constants.h"
+#include "brave/components/brave_wallet/browser/buildflags/buildflags.h"
+
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
```

Comment:
this should probably be `ENABLE_EXTENSIONS` I think since this header isn't brave wallet specific

Reply:
I was going back and forth trying to think of which way was better, I went with wallet since it could be changed later if it needed to be included in another case. But I'll change to `ENABLE_EXTENSIONS`. Fixed.